docs/howto/system_testing.md

```diff
@@ -66,7 +79,42 @@ services:
 
 Here, `SERVICE_LOGS_DIR` is a special keyword. It is something that we will need later.
 
-### Data stream-level configuration
+### Terraform service deployer
+
+The `*.tf` files define the infrastructure using the Terraform syntax. The terraform based service can be handy to boot up
```

Comment:
Similar to my comment on the Docker Composer service deployer, I would add an introductory sentence to bridge the gap between what's shown in the sample file structure and this sentence here.

Reply:
Fixed